src/modules/auth/controllers/auth.controller.ts

```
@@ -1,33 +1,53 @@
-
 import { AuthService } from '../services/auth.service';
 import { RegisterDto } from '../dto/register.dto';
 import { LoginDto } from '../dto/login.dto';
+import { Request } from 'express';
+import { promisify } from 'util';
+import { JwtAuthGuard } from '../guards/jwt-auth.guard';
 import { User } from '../entities/user.entity';
```

ESLint

[error] 'User' is defined but never used.

(@typescript-eslint/no-unused-vars)